src/Sentry/User.cs

```diff
@@ -11,37 +12,79 @@ namespace Sentry
     /// <see href="https://develop.sentry.dev/sdk/event-payloads/user/"/>
     public sealed class User : IJsonSerializable
     {
+        internal Action? PropertyChanged { get; set; }
```

Comment:
Maybe this I mean:
```suggestion
        internal Action<User>? PropertyChanged { get; set; }
```

So you pass `PropertyChanged(this)`